### PR TITLE
Patch validator

### PR DESCRIPTION
**Pull request by velibor7**
- Implemented patch validator method
- Implemented tests

### CODE REVIEW

**Comment by velibor7** [comment]
Relative issue: #22

File: tnb/banks.py [hunk 1]
```diff
@@ -53,6 +53,25 @@ def patch_account(self, account_number, node_id, trust, signature):
 
         return self.patch(resource, body=body)
 
+    def patch_validator(self, node_id, trust, signature):
+        """
+        Send a PATCH request of an account to a Bank
+
+        :param node_identifier: Node identifier of the bank
```
Comment (shahraizali) [inline tnb/banks.py:60]: let's keep the comments consistent with the naming convention. Also, I think you are missing some params

Comment (velibor7) [reply tnb/banks.py]: Off, my bad. Fixed it with d4f6220

File: tests/banks.py [hunk 2]
```diff
@@ -124,6 +124,38 @@ def test_success_patch_account(requests_mock):
     assert response == result
 
 
+def test_success_patch_validator(requests_mock):
```
Comment (sbalasa) [inline tests/banks.py:127]: Please add docstrings to this function

**Review by sbalasa — APPROVED** [review]

**Pull request merged**